core/src/main/scala/org/scalarelational/instruction/Query.scala

```diff
@@ -55,4 +59,43 @@ case class Query[Expressions, Result](expressions: Expressions,
   def async = table.datastore.async {
     result
   }
+
+  def to[R](implicit manifest: Manifest[R]) = {
```

Comment:
I really don't like the Mapper functionality being integrated into `core` like this. My hope is to keep `core` clean and SQL focused while enabling wrapper functionality to add more powerful functionality like this.